google/cloud/security/inventory/pipelines/load_projects_cloudsql_pipeline.py

```diff
@@ -34,6 +34,7 @@ class LoadProjectsCloudsqlPipeline(base_pipeline.BasePipeline):
     """Pipeline to load project CloudSql data into Inventory."""
 
     PROJECTS_RESOURCE_NAME = 'project_iam_policies'
+    RESOURCE_NAME = 'cloudsql'
```

Comment:
As a long term thing, would it make sense to move the resource names as keys under the requirements map?